README.md

```diff
@@ -73,7 +73,7 @@ A secret identity.
 The server-side pseudo-code below demonstrates a typical flow to safely deliver identities to your users:
 
 ```ruby
-require 'tanker/identity'
+require 'tanker-identity'
```

Comment:
you could leave that, I guess

Reply:
Both forms are acceptable in the end, we can document any... I preferred to keep the old form so that people won't think they have to migrate.